module.json

```diff
@@ -44,6 +44,9 @@
     },
     "arm-ssg": {
       "mbed-hal-arm-ssg": "^1.0.0"
+    },
+    "atmel": {
+      "mbed-hal-atmel": "^0.1.0"
```

Comment:
From the link you referenced 
> ~1.2.3: any version with the same major and minor versions, and an equal or greater patch version.

this is what you want to achieve here for module with 0.x.x . Why mbed-hal-atmel is 0.x.x , not 1.x.x?